test/helpers_test.rb

```diff
@@ -879,6 +879,12 @@ def send_file_app(opts={})
       assert_equal 'inline; filename="file.txt"', response['Content-Disposition']
     end
 
+    it "does not raise an error when :disposition set to a frozen string" do
+      send_file_app :disposition => 'inline'.freeze
```

Comment:
@zzak It happened to up when we added the magic frozen string literal comment at the top of our files. 

Since we had `inline` as a literal in the file, it was frozen and then the in attachment method we get an error modifying a frozen string literal.